lib/src/css_parser.dart

```diff
@@ -70,6 +115,266 @@ class ExpressionMapping {
     return null;
   }
 
+  static TextDirection expressionToDirection(css.Expression value) {
+    if (value is css.LiteralTerm) {
+      switch(value.text) {
+        case "ltr":
+          return TextDirection.ltr;
+        case "rtl":
+          return TextDirection.rtl;
+      }
+    }
+    return TextDirection.ltr;
+  }
+
+  static Display expressionToDisplay(css.Expression value) {
+    if (value is css.LiteralTerm) {
+      switch(value.text) {
+        case 'block':
+          return Display.BLOCK;
+        case 'inline-block':
+          return Display.INLINE_BLOCK;
+        case 'inline':
+          return Display.INLINE;
+        case 'list-item':
+          return Display.LIST_ITEM;
+        case 'none':
+          return Display.NONE;
+      }
+    }
+    return Display.INLINE;
+  }
+
+  static List<FontFeature> expressionToFontFeatureSettings(List<css.Expression> value) {
+    List<FontFeature> fontFeatures = [];
+    for (int i = 0; i < value.length; i++) {
+      css.Expression exp = value[i];
+      if (exp is css.LiteralTerm) {
+        if (exp.text != "on" && exp.text != "off" && exp.text != "1" && exp.text != "0") {
+          if (i < value.length - 1) {
+            css.Expression nextExp = value[i+1];
+            if (nextExp is css.LiteralTerm && (nextExp.text == "on" || nextExp.text == "off" || nextExp.text == "1" || nextExp.text == "0")) {
+              fontFeatures.add(FontFeature(exp.text, nextExp.text == "on" || nextExp.text == "1" ? 1 : 0));
+            } else {
+              fontFeatures.add(FontFeature.enable(exp.text));
+            }
+          } else {
+            fontFeatures.add(FontFeature.enable(exp.text));
+          }
+        }
+      }
+    }
+    fontFeatures.toSet().toList();
```

Comment:
This doesn't do anything right? (it creates a new list, bu you don't assign it to anything).

Reply:
It removes duplicates from the list. Kind of odd, but it works nicely and is super concise.

Reply:
I thought that's what you wanted, but it doesn't, because the list is not modified in-place.
```dart
    test("toSet_inline", () async {
      final list = ["one", "two", "one"];
      list.toSet().toList();
      expect(list, ["one", "two"]);
    });
```
Will fail:
```
Expected: ['one', 'two']
  Actual: ['one', 'two', 'one']
```

Reply:
I thought it was modified in place. I will fix this as well.

Reply:
Fixed.